### ISSUE TITLE
NOOB I want to give this a try, But I dont seem to understand the instructions

### ISSUE THREAD

**Issue by S0L0d0b0** [jdah/minecraft-weekend#77]
Im not well educated in coding but I know a little. Unfortunately not well enough to achieve opening the game. I would just like some help please

**Comment by TheTechRobo** [comment]
We can't help you if you don't tell us your problem. What don't you understand?

**Comment by S0L0d0b0** [comment]
so I did the first command 
$ git clone --recurse-submodules https://github.com/jdah/minecraft-weekend.git

the next command is $ make
Idk if im supposed to know  if there's something supposed to be after that, like $ make... ?

The following static libraries under lib/ must be built before the main project can be built:

    GLAD lib/glad/src/glad.o
    CGLM lib/cglm/.libs/libcglm.a
    GLFW lib/glfw/src/libglfw3.a
    libnoise lib/noise/libnoise.a

so im guessing im supposed to make the files: glad.o, libcglm.a, libglfw3.a, libnoise.a,   because they aren't there.
what I did was I used the make function as   make glad.o    while in the subdirectory 

I got 
cc    -c -o glad.o glad.c
glad.c:633:10: fatal error: glad/glad.h: No such file or directory
  633 | #include <glad/glad.h>
      |          ^~~~~~~~~~~~~
compilation terminated.
make: *** [<builtin>: glad.o] Error 1
Note Im not familiar with a makefile.  I dont know what it is.

All of the above have their own Makefile under their respective subdirectory and can be built with $ make libs. If libraries are not found, ensure that submodules have been cloned.

are the "makefiles under their respective subdirectory" the same as the "static libraries under lib/ that must be built"? is he talking about the same thing?  
I went to the subdirectory lib/glad/src and on terminal I typed " make libs, make glad.o, make libs glad.o, wich I got nothing for any of it.   
So yea I dont know what is going on. heck I dont know what to even explain what I dont understand because I just dont understand any of it.   make sence?

**Comment by 0rbianta** [comment]
Some of the libraries not exist at ./lib. You should clone cglm and glad from github to ./lib location.
After installing them build the libraries first before building the game so it can link them.
Clone libraries by:
git clone https://github.com/recp/cglm.git
...

Build libraries:
make libs
Make bin directory:
make dirs
Build game:
make game
Run game:
make run

It might give a duplicated definition error due to library versions not same. Remove one defined at ./src if that happens

**Comment by valvhammer** [comment]
```console
$ make libs

$ make 
```

should be enough

**Comment by avighnac** [comment]
Yes, but note that you might need to install `libxcursor`, `libxinerama`, and stuff like that. You can google how to download those libraries for your linux distro.
